crates/ruff_python_formatter/src/expression/expr_tuple.rs

```diff
@@ -111,12 +114,12 @@ impl FormatNodeRule<ExprTuple> for FormatExprTuple {
             // Unlike other expression parentheses, tuple parentheses are part of the range of the
             // tuple itself.
             _ if is_parenthesized(*range, elts, f.context().source())
-                && self.parentheses != TupleParentheses::StripInsideForLoop =>
+                && self.parentheses != TupleParentheses::NeverWithoutReason =>
```

Comment:
I'm not in love with this.

Reply:
Maybe ec9b23d4e0a041c0c150d789e4e32be9784ce96f

Reply:
But I guess same question as https://github.com/astral-sh/ruff/pull/5790#discussion_r1264593822 but for `TupleParentheses::Expr(Parentheses::Preserve)`.

Reply:
d7f0c26a3ac870caba44b80a0622adbb217ecbc2